contracts/lending/WasabiBNPL.sol

```diff
@@ -59,6 +59,24 @@ contract WasabiBNPL is IWasabiBNPL, Ownable, IERC721Receiver, ReentrancyGuard {
         factory = _factory;
     }
 
+    /// @dev Returns the option data for the given option id
+    function getOptionData(
+        uint256 _optionId
+    ) external view returns (WasabiStructs.OptionData memory optionData) {
+        LoanInfo memory loanInfo = optionToLoan[_optionId];
+        INFTLending.LoanDetails memory loanDetails = INFTLending(
+            loanInfo.nftLending
+        ).getLoanDetails(loanInfo.loanId);
+
+        optionData = WasabiStructs.OptionData(
+            true,
```

Comment:
For the active parameter, lets check:
* loan expiration
* the owner of the option is not address(0)